Frontend/ui-customer/src/Flow.purs

```diff
@@ -542,6 +542,34 @@ homeScreenFlow = do
       modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen{props{searchId = rideSearchRes.searchId,currentStage = FindingEstimate, rideRequestFlow = true, isSearchLocation = SearchLocation, sourcePlaceId = Nothing, destinationPlaceId = Nothing}})
       updateLocalStage FindingEstimate
       homeScreenFlow
+    RETRY_FINDING_QUOTES -> do
+      void $ lift $ lift $ loaderText (getString LOADING) (getString PLEASE_WAIT_WHILE_IN_PROGRESS)  -- TODO : Handlde Loader in IOS Side
+      void $ lift $ lift $ toggleLoader true
+      (GlobalState newState) <- getState
+      let state = newState.homeScreen
+      if (not (isLocalStageOn QuoteList)) then do
+        _ <- pure $ firebaseLogEvent "ny_user_cancel_and_retry_request_quotes"
+        cancelEstimate state.props.estimateId
+      else do
+        _ <- pure $ firebaseLogEvent "ny_user_retry_request_quotes"
+        pure unit
+      _ <- pure $ setValueToLocalStore AUTO_SELECTING "false"
```

Comment:
why does this function has a bind operation required but same func in next line don't?